test/benchmarks/mdc/tabs/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+load("@npm_angular_dev_infra_private//benchmark/component_benchmark:component_benchmark.bzl", "component_benchmark")
+
+# TODO(wagnermaciel): Update this target to provide indigo-pink in a way that doesn't require having to import it with
+# stylesUrls inside the components once `component_benchmark` supports asset injection.
+
+component_benchmark(
+    name = "benchmark",
+    driver = ":tabs.perf-spec.ts",
+    driver_deps = [
+        "@npm//@angular/dev-infra-private",
+        "@npm//protractor",
+        "@npm//@types/jasmine",
+    ],
+    ng_deps = [
+        "@npm//@angular/core",
+        "@npm//@angular/platform-browser",
+        "//src/material-experimental/mdc-tabs",
+    ],
+    ng_srcs = [
+        ":app.module.ts",
+        ":fake-tab-data.ts",
```

Comment:
Just a FYI: The usual convention is that colon is only used to indicate a Bazel target. Plain input files are without the colon.